topics/cluster-spec.md

```diff
@@ -470,20 +470,20 @@ is that:
 * All queries about non-existing keys in A are processed by "B", because "A" will redirect clients to "B".
 
 This way we no longer create new keys in "A".
-In the meantime, a special program called `redis-trib` used during reshardings
+In the meantime, a special program called `redis-cli` used during reshardings
 and Redis Cluster configuration will migrate existing keys in
 hash slot 8 from A to B.
```

Comment:
Is `redis-cli` a "special" program? I think it is the normal CLI client shipped with Redis...

Reply:
Remove it.